Dockerfile.testrpc

```diff
@@ -0,0 +1,5 @@
+FROM node:8-alpine
+RUN npm install -g ethereumjs-testrpc
+COPY build/testrpc_persist /data/persist
```

Comment:
You're not adding the shell script in here to automatically migrate and save the contracts?